File: java/javadpbo/src/com/sem4dpbo/week2/contohkasus/Kerajinan.java
package com.sem4dpbo.week2.contohkasus;

public class Kerajinan extends HasilKerajinan {
    private int harga;
    private String fungsi;

    Kerajinan(){}

    public void setHarga(int harga) {
        this.harga = harga;
    }

    public void setFungsi(String fungsi) {
        this.fungsi = fungsi;
    }

    public int getHarga() {
        return harga;
    }

    public String getFungsi() {
        return fungsi;
    }
}

File: cpp/2nd_Week/ContohKasusKayu/Kerajinan.cpp
#include "HasilKerajinan.cpp"

class Kerajinan: public HasilKerajinan
{
    private:
        /* data */
        int harga;
        string fungsi;
    public:
        Kerajinan(){};

        void setHarga(int harga) {
            this->harga = harga;
        }

        void setFungsi(string fungsi) {
            this->fungsi = fungsi;
        }

        int getHarga() {
            return harga;
        }

        string getFungsi() {
            return fungsi;
        }

        ~Kerajinan(){};
};
